### PR TITLE
Fix EmbedBuilder constructor javadocs

### PR DESCRIPTION
**Pull request by caneleex**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [ ] Library interface (affecting end-user code) 
- [x] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

This PR fixes and improves EmbedBuilder constructor javadocs.

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/api/EmbedBuilder.java [hunk 1]
```diff
@@ -24,7 +25,7 @@
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
-import java.awt.Color;
+import java.awt.*;
```
Comment (Andre601) [inline src/main/java/net/dv8tion/jda/api/EmbedBuilder.java]: I don't think it's good to have java.awt.* here.

File: src/main/java/net/dv8tion/jda/api/EmbedBuilder.java [hunk 2]
```diff
@@ -56,7 +57,7 @@
     private MessageEmbed.ImageInfo image;
 
     /**
-     * Creates an EmbedBuilder to be used to creates an embed to send.
+     * Constructs a new EmbedBuilder instance, which can be used to create {@link MessageEmbed MessageEmbeds}. These can then be sent to a channel using {@link MessageChannel#sendMessage(MessageEmbed)}.
```
Comment (Andre601) [inline src/main/java/net/dv8tion/jda/api/EmbedBuilder.java]: Shouldn't the links be the full path?

File: src/main/java/net/dv8tion/jda/api/EmbedBuilder.java (hunk 1 above)
Comment (caneleex) [reply src/main/java/net/dv8tion/jda/api/EmbedBuilder.java]: Fixed.

File: src/main/java/net/dv8tion/jda/api/EmbedBuilder.java (hunk 2 above)
Comment (caneleex) [reply src/main/java/net/dv8tion/jda/api/EmbedBuilder.java]: Fixed.

**Pull request merged**